pkg/apis/kops/cluster.go

```diff
@@ -92,6 +92,8 @@ type ClusterSpec struct {
 	NonMasqueradeCIDR string `json:"nonMasqueradeCIDR,omitempty"`
 	// SSHAccess is a list of the CIDRs that can access SSH.
 	SSHAccess []string `json:"sshAccess,omitempty"`
+	// NodePortAccess is a list of the CIDRs that can access the node ports range (30000-32767).
+	NodePortAccess []string `json:"nodePortAccess,omitempty"`
```

Comment:
nit pick, but the name does not make sense to me ...  It is an API value so we need to have it named well.  @blakebarnett / @alrs / @KashifSaadat  thoughts?  NodePortAccess sounds like a true / false. Not a list of CIDRS

NodePortSecGroupRanges?

Reply:
It is supposed to be a parallel to kubernetesApiAccess and sshAccess, but open to better ideas.

Reply:
That makes sense.